app/account_linking.py

```diff
@@ -38,6 +51,38 @@ class LinkResult:
     strategy: str
 
 
+def send_double_subscription_if_needed(partner_user: PartnerUser):
+    sub = partner_user.user.get_active_subscription()
+    if isinstance(sub, CoinbaseSubscription):
+        channel = "Coinbase"
+    elif isinstance(sub, AppleSubscription):
+        channel = "Apple"
+    elif isinstance(sub, Subscription):
+        channel = "Paddle"
+    else:
+        return
+    send_email_at_most_times(
+        partner_user.user,
+        config.ALERT_DUAL_SUBSCRIPTION_WITH_PARTNER,
+        partner_user.user.email,
+        f"You have two subscriptions in SimpleLogin",
+        render(
+            "transactional/double-subscription-partner.txt.jinja2",
+            is_proton=is_proton_partner(partner_user.partner),
+            partner=partner_user.partner,
+            subscription_channel=channel,
+            cancel_link=url_for("dashboard.billing"),
+        ),
+        render(
+            "transactional/double-subscription-partner.html",
+            is_proton=is_proton_partner(partner_user.partner),
+            partner=partner_user.partner,
+            subscription_channel=channel,
+            cancel_link=url_for("dashboard.billing"),
```

Comment:
the cancel link here is only for paddle subscription actually. For apple subscription, it's to be done in user icloud account.

Reply:
Could me show that info in the billing template? If you have apple put a notice that to modify the billing you have to go to apple. And if you've got coinbase then you can't do jack. But we can show until when the sub is active in any case, even if you have a partner sub.

Reply:
I think it might be easier to distinguish this in the email. For apple user, we can tell them to unsubscribe in their iCloud account. We can also optionally send them this link https://support.apple.com/en-us/HT202039

Reply:
Ok. I'll change it.